contracts/LivepeerProtocol.sol

```diff
@@ -121,6 +121,22 @@ contract LivepeerProtocol {
     // rewardMultiplier[1] -> transcoder's cumulative stake for round
     mapping (address => mapping (uint256 => uint256[2])) public rewardMultiplierPerTranscoderAndRound;
 
+    // The various states a job can be in
+    enum JobStatus { Inactive, Unclaimed, Claimed, Verified }
```

Comment:
There's a state that I'm not sure how to handle...

I think a transcoder should be able to call `endJob()` for any range of segments, which means that they may make **many** calls to `endJob()` for a particular job. For example...if they never received segment 9, they may call `endJob(1, 8)` and then `endJob(10, 99)`. 

Perhaps it's possible that a job never truly ends, and it's only the claims that are closed out, based on the time limit from when a claim is submitted until it is verified.